config/settings.yml

```diff
@@ -267,7 +265,10 @@ saml:
   issuer: saml-rp.vetsgov.localhost
   callback_url: http://localhost:3000/auth/saml/callback
   metadata_url: https://api.idmelabs.com/saml/metadata/provider
-  relay: http://localhost:3001/auth/login/callback
+  relay:
+    original: http://localhost:3001/auth/login/callback
+    alternate: http://doesntexist
+    review_instance_slug: ~
   logout_relay: http://localhost:3001/logout
```

Comment:
Are there better names we could use other than "original" "alternate" etc? That better communicate what the purpose of these are? I think "original" and "alternate" imply a certain context that will no longer exist 2-3 months from now.

Reply:
Agreed and responded in another comment, let's discuss there.